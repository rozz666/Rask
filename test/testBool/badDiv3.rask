main() -> void
{
    var x = 7 / false;
}