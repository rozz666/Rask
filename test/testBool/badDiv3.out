badDiv3.rask:3:15: error: function binary/(int32, boolean) not found
